core/payment-driver/zksync/src/driver.rs

```diff
@@ -0,0 +1,131 @@
+/*
+    ZksyncDriver to handle payments on the zksync network.
+
+    Please limit the logic in this file, use local mods to handle the calls.
+*/
+// Extrnal crates
+use uuid::Uuid;
+
+// Workspace uses
+use ya_payment_driver::{
+    account::{AccountsRc, AccountsRefMut},
+    bus,
+    driver::{async_trait, BigDecimal, PaymentDriver},
+    model::{
+        Ack, GenericError, GetAccountBalance, GetTransactionBalance, Init, PaymentDetails,
+        SchedulePayment, VerifyPayment,
+    },
+    utils as driver_utils,
+};
+
+// Local uses
+use crate::{zksync::wallet, DRIVER_NAME, PLATFORM_NAME};
+
+pub struct ZksyncDriver {
+    active_accounts: AccountsRc,
+}
+
+impl ZksyncDriver {
+    pub fn new(accounts: AccountsRc) -> Self {
```

Comment:
Any legit cases when we'll be passing non-empty `accounts` to the constructor?

Reply:
When fetching accounts on startup it could.
But then this could also fetch the accounts from the bus

relates to https://golemproject.atlassian.net/browse/PAY-113

Reply:
I just thought it might have a param-less constructor. But also we can `#[derive(Default)]` (on `AccountsRc` as well then).